server/odc-service/src/main/java/com/oceanbase/odc/service/task/schedule/daemon/DoCancelingJob.java

```diff
@@ -81,15 +82,15 @@ private void cancelJob(TaskFrameworkService taskFrameworkService, JobEntity jobE
                     throw new TaskRuntimeException(
                             "Update job status to CANCELED failed, jobId=" + lockedEntity.getId());
                 }
-                log.info("Prepare cancel task, jobId={}.", lockedEntity.getId());
-                try {
-                    getConfiguration().getJobDispatcher().stop(JobIdentity.of(lockedEntity.getId()));
-                } catch (JobException e) {
-                    log.warn("Stop job occur error: ", e);
-                    AlarmUtils.alarm(AlarmEventNames.TASK_CANCELED_FAILED,
-                            MessageFormat.format("Cancel job failed, jobId={0}", lockedEntity.getId()));
-                    throw new TaskRuntimeException(e);
-                }
+                // log.info("Prepare cancel task, jobId={}.", lockedEntity.getId());
```

Comment:
if do not stop job here, what is the new location?

Reply:
StdJobScheduler.tryCanceling